internal/pkg/mining/scheduler_test.go

```diff
@@ -6,154 +6,17 @@ import (
 	"testing"
 	"time"
 
-	"github.com/filecoin-project/go-address"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
 	"github.com/filecoin-project/go-filecoin/internal/pkg/block"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/chain"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/clock"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/consensus"
 	e "github.com/filecoin-project/go-filecoin/internal/pkg/enccid"
 	. "github.com/filecoin-project/go-filecoin/internal/pkg/mining"
-	th "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers"
 	tf "github.com/filecoin-project/go-filecoin/internal/pkg/testhelpers/testflags"
 	"github.com/filecoin-project/go-filecoin/internal/pkg/types"
-	"github.com/filecoin-project/go-filecoin/internal/pkg/vm/state"
 )
 
-// TestMineOnce10Null calls mine once off of a base tipset with a ticket that
-// will win after 10 rounds and verifies that the output has 1 ticket and a
-// +10 height.
-func TestMineOnce10Null(t *testing.T) {
```

Comment:
Ideally we would keep one of these tests.  However they have been skipped for a month now because of vm switch technical debt we haven't yet addressed.  This PR just digs us deeper in the tech debt hole and rather than comment it out or fake out SeedFirstWinnerInNRounds I'm removing with the hope that we can add something like this back during painnet tech debt pay down.